default/scripting/ship_hulls/monster/SH_IMMOBILE_FACTOR.focs.txt

```diff
@@ -15,7 +15,7 @@ Hull
     Unproducible
     location = All
     effectsgroups = [
-        [[GOOD_VISION]]
+        [[UNOWNED_OWNED_VISION_BONUS(WEAK,15,10)]]
```

Comment:
rebased.  dropped to 10 here and raised to 40 for Acirema/Sentinel (catching +stealth organics without cloak)